cmd/scale.go

```diff
@@ -16,24 +16,24 @@ var (
 			cli.BoolFlag{Name: "synchronous, s", Usage: "Do the scaling synchronously", EnvVar: ""},
 		},
 		Usage: "Scale your application instantly",
-		Description: `Scale your application processes.
+		Description: `Scale your application processes. Without argument, this command lists the container types declared in your application.
    Example
      'scalingo --app my-app scale web:2 worker:1'
      'scalingo --app my-app scale web:1 worker:0'
      'scalingo --app my-app scale web:1:XL'
      'scalingo --app my-app scale web:+1 worker:-1'
      `,
 		Action: func(c *cli.Context) {
+			currentApp := appdetect.CurrentApp(c)
+
 			if len(c.Args()) == 0 {
-				err := cli.ShowCommandHelp(c, "scale")
+				err := apps.Containers(currentApp)
```

Comment:
Could you rename `apps.ContainerTypes()` and match this nomenclature which is more accurate about the name of our resources?